packages/apple-pay-integration/src/ApplePayPaymentMethod.spec.tsx

```diff
@@ -1,17 +1,17 @@
-import { getMethod, PaymentMethodId, PaymentFormService } from "@bigcommerce/checkout-js/payment-integration";
+import { getMethod, PaymentFormService } from "@bigcommerce/checkout-js/payment-integration";
 import { createCheckoutService, LanguageService } from "@bigcommerce/checkout-sdk";
 import { mount } from 'enzyme';
 import React from "react";
 
-import ApplePaymentMethod from './ApplePayPaymentMethod';
+import ApplePaymentMethod, {resolveId} from './ApplePayPaymentMethod';
 
 describe('ApplePay payment method', () => {
     const checkoutService = createCheckoutService();
     const checkoutState = checkoutService.getState();
     const props = {
         method: {
             ...getMethod(),
-            id: PaymentMethodId.ApplePay,
+            id: resolveId,
```

Comment:
I think just `id: 'applepay'` is suffice. See https://github.com/bigcommerce/checkout-js/pull/954/files#diff-ee22dc27a6eef16f03b848a02e78862f21c21902a608373c0caf6340dc98f305R14

Reply:
@davidchin Thanks, DC.